Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/calibrations.py

```diff
@@ -0,0 +1,80 @@
+"""
+calibrations.py: template for easily performing single qubit calibration protocols.
+"""
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+from qualang_tools.addons.calibration.calibrations import QUA_calibrations
+
+
+# Relevant configuration parameters
+#############################################################
+resonator_element = "resonator"
+resonator_operation = "readout"
+qubit_element = "qubit"
+qubit_operation = "gauss"
+flux_line_element = "flux_line"
+flux_line_operation = "const"
+int_w = ["cos", "sin", "minus_sin", "cos"]
+outs = ["out1", "out2"]
+#############################################################
+
+# Initialize the calibration API with the relevant configuration parameters
+my_calib = QUA_calibrations(
+    configuration=config,
+    readout=(resonator_element, resonator_operation),
+    qubit=(qubit_element, qubit_operation),
+    integration_weights=int_w,
+    outputs=outs,
+)
+
+n_avg = 1000
+# Resonator spectroscopy scan
+scans = [
+    ("frequency", np.arange(100e6, 300e6, step=0.1e6)),
+]
+my_calib.set_resonator_spectroscopy(scan_variables=scans, iterations=n_avg, cooldown_time=0)
+
+# Rabi scan
+scans = [
+    ("duration", np.arange(40, 4000, step=100)),
+    ("amplitude", np.linspace(0.5, 1.99, num=51)),
+]
+# my_calib.set_rabi(scan_variables=scans, iterations=n_avg, cooldown_time=0)
+
+# T1 scan
+scans = [
+    ("duration", np.arange(5, 4000, step=10)),
+]
```

Comment:
```suggestion
scans = [("duration", np.arange(5, 4000, step=10))]
```
If you remove the ending comma, than the black formatter won't expand it and it'll be more compact